### PR TITLE
Add max player health decrease

### PR DESCRIPTION
**Pull request by leo-210**

### CODE REVIEW

File: src/main/java/world/anhgelus/architectsland/difficultydeathscaler/DifficultyDeathScaler.java [hunk 1]
```diff
@@ -57,58 +72,90 @@ public void run() {
...
     private void decreaseDeath(MinecraftServer server) {
-        if (numberOfDeath >= 7) {
-            numberOfDeath = 5;
-        } else if (numberOfDeath >= 5) {
-            numberOfDeath = 3;
-        } else if (numberOfDeath >= 3) {
-            numberOfDeath = 1;
-        } else if (numberOfDeath >= 1) {
-            numberOfDeath = 0;
+        for (int i = deathSteps.length - 1; i >= 0; i--) {
```
Comment (anhgelus) [inline src/main/java/world/anhgelus/architectsland/difficultydeathscaler/DifficultyDeathScaler.java:75]: Use a for each:
```java
for (int step : deathSteps) {
    if (numberOfDeath >= step) {
        numberOfDeath = step;
        break;
    }
}
```

Comment (anhgelus) [reply src/main/java/world/anhgelus/architectsland/difficultydeathscaler/DifficultyDeathScaler.java:75]: I guess I don't see it was a reverse for each...

**Review by anhgelus — APPROVED** [review]

**Comment by anhgelus** [comment]
I'm waiting to the end of the poll to merge.

**Comment by leo-210** [comment]
I will test the death decrease, I just found something that needed to be fixed (i.e. latest commit)

**Comment by leo-210** [comment]
Ok it works perfectly

**Comment by leo-210** [comment]
my bad there was one more problem

**Review by anhgelus — APPROVED** [review]

**Pull request merged**